tests/test_to_hepmc3.py

```diff
@@ -24,24 +27,19 @@ def run(Model):
 
 @pytest.mark.parametrize("Model", models)
 def test_to_hepmc3(Model):
-    if Model == im.UrQMD34:
-        pytest.xfail("UrQMD34 FAILS, should be FIXED!!!")
+    if Model.name in ("PhoJet", "DPMJET-III"):
```

Comment:
Is this really the case? Pythia8 should also have an altered event stack.

Reply:
Below, there were a Pythia8 specific code, therefore the test passed. The code in the test has been changed to use hepmc fix.